### PR TITLE
[THOM CUP] End of the tournament update

### PR DESCRIPTION
**Pull request by WalterToro**
- Added prizes and podium.
- Added finals and grand finals results
- Added grandfinals mappool
- Changed /b/ to /beatmaps/ on links

### CODE REVIEW

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 1]
```diff
@@ -210,6 +233,19 @@ The Heptakey osu!mania Cup #1
...
 
+### Finals
+
+| Saturday, 2018-07-07 | | | | |
+| ---: | :---: | :---: | :--- | :---: |
+| Keweft ![][flag_BR]      | 0 | **7** | ![][flag_IT] **Yomiel**  | [#1](https://osu.ppy.sh/community/matches/43968717) |
+| **aitor98** ![][flag_ES] | **7** | 1 | ![][flag_CL] Skalim      | [#1](https://osu.ppy.sh/community/matches/43977039) |
+| Lothus ![][flag_BR]      | 0 | **7** | ![][flag_US] **Gekido-** | -Win by Default- |
+
+
```
Comment (Joehuu) [inline wiki/Tournaments/THOM_CUP/1/en.md]: Remove the extra line.

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 2]
```diff
@@ -78,49 +78,72 @@ The Heptakey osu!mania Cup #1
...
+**[Download the mappack here! (97.3 MB)](https://www.dropbox.com/sh/dhys8odeifwlom8/AAAyH8eQmAoWvUFdwNJC91DRa?dl=0)**
+
+- FreeMod
+  - [penoreri - Sailing Force (AncuL) \[Weigh Anchor\]](https://osu.ppy.sh/beatmaps/1491970)
+  - [Camellia - Illegal trap (Flexo123) \[EX\]](https://osu.ppy.sh/beatmaps/1592340)
+  - [Hino Isuika - Dreamin' attraction!!(Extended) (MapleSyrup-) \[Orange Dream\]](https://osu.ppy.sh/beatmaps/1370533)
+  - [Miyu Tomita & Mariya Ise - Deep in Abyss (LostCool) \[Have you ever seen a dawn?\]](https://osu.ppy.sh/beatmaps/1400630)
+  - [Camellia - ANOMALY (qodtjr) \[The Last Madness\]](https://osu.ppy.sh/beatmaps/1285947)
+  - [movies - Realize Maze (Extended Mix) (moni) \[Circuitousness\]]    (https://osu.ppy.sh/beatmaps/791775)
```
Comment (Joehuu) [inline wiki/Tournaments/THOM_CUP/1/en.md]: Broken link. Remove the spaces.

**Comment by WalterToro** [comment]
Updated PR to add final results for the tournament.

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ The Heptakey osu!mania Cup #1
 
 ![THOM CUP Logo](logo.png)
 
-**The Heptakey osu!mania Cup \#1** (***THOM CUP \#1***) is a one-on-one euro-american osu!mania tournament. It is the 1st installment of the THOM CUP. More specifically, this tournament is an osu!mania 7-key tournament.
+**The Heptakey osu!mania Cup \#1** (***THOM CUP \#1***) was a one-on-one euro-american osu!mania tournament. It was the 1st installment of the THOM CUP. More specifically, this tournament is an osu!mania 7-key tournament.
```
Comment (TPGPL) [inline wiki/Tournaments/THOM_CUP/1/en.md]: no reason to change that tbh

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 4]
```diff
@@ -28,7 +28,11 @@ The Heptakey osu!mania Cup #1
 
 ## Prizes
 
-*The prizes are to be decided.*
+| Placing | Prize |
+| :-: | :--- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | $50 USD |
```
Comment (TPGPL) [inline wiki/Tournaments/THOM_CUP/1/en.md]: choose either $ or USD i think

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 5]
```diff
@@ -76,51 +80,88 @@ The Heptakey osu!mania Cup #1
...
-  - [p_d - Dandelion Sparkle!! (qodtjr) \[Sprint\]](https://osu.ppy.sh/b/1649634)
-  - [IU - You & I (moni) \[I\]](https://osu.ppy.sh/b/824900)
-  - [void - Sudden Romance (\_underjoy) \[Chordmancer (short)\]](https://osu.ppy.sh/b/1597751)
+  - [Kyary Pamyu Pamyu - Mottai Night Land (Cuppp) \[Insane\]](https://osu.ppy.sh/beatmaps/691829)
+  - [wa. - Black Lotus (Rain-) \[CuCu\]](https://osu.ppy.sh/beatmaps/1436278)
+  - [fripSide - infinite synthesis (\_underjoy) \[Benzopirene's LN MADNESS (cut)\]](https://osu.ppy.sh/beatmaps/1693911)
+  - [ginkiha - Anemoi (Ramuchi) \[Spring Wind\]](https://osu.ppy.sh/beatmaps/1473413)
+  - [w_tre - FIONA (stupud man) \[Hi (rework)\]](https://osu.ppy.sh/beatmaps/1694310)
+  - [Round Wave Crusher - Slash Dot Slash (Slim boy fat) (Evening) \[destinia vs. estear\]](https://osu.ppy.sh/beatmaps/1677804)
+  - [PLight - TRIGGER*HAPPY (Hommarju Remix) (\_underjoy) \[MAXIMUM\]](https://osu.ppy.sh/beatmaps/1498431)
```
Comment (TPGPL) [inline wiki/Tournaments/THOM_CUP/1/en.md]: escape the `*`

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 6]
```diff
@@ -76,51 +80,88 @@ The Heptakey osu!mania Cup #1
...
+  - [Kyary Pamyu Pamyu - Mottai Night Land (Cuppp) \[Insane\]](https://osu.ppy.sh/beatmaps/691829)
+  - [wa. - Black Lotus (Rain-) \[CuCu\]](https://osu.ppy.sh/beatmaps/1436278)
+  - [fripSide - infinite synthesis (\_underjoy) \[Benzopirene's LN MADNESS (cut)\]](https://osu.ppy.sh/beatmaps/1693911)
+  - [ginkiha - Anemoi (Ramuchi) \[Spring Wind\]](https://osu.ppy.sh/beatmaps/1473413)
+  - [w_tre - FIONA (stupud man) \[Hi (rework)\]](https://osu.ppy.sh/beatmaps/1694310)
+  - [Round Wave Crusher - Slash Dot Slash (Slim boy fat) (Evening) \[destinia vs. estear\]](https://osu.ppy.sh/beatmaps/1677804)
+  - [PLight - TRIGGER\*HAPPY (Hommarju Remix) (\_underjoy) \[MAXIMUM\]](https://osu.ppy.sh/beatmaps/1498431)
+  - [3L - Endless night (Fiea) \[deyum\]](https://osu.ppy.sh/beatmaps/455082)
+  - [Venetian Snares - Ketsarku Mozgalom (wizardoffail) \[only in comparison\]](https://osu.ppy.sh/beatmaps/766440)
+  - [litmus* - iterated (edit) (Kamikaze) \[7K INFINITE\]](https://osu.ppy.sh/beatmaps/960117)
```
Comment (TPGPL) [inline wiki/Tournaments/THOM_CUP/1/en.md]: here too

**Pull request merged**